src/actions/TransactionExportActions.tsx

```diff
@@ -21,6 +26,36 @@ export async function exportTransactionsToCSV(wallet: EdgeCurrencyWallet, txs: E
   return exportTransactionsToCSVInner(txs, currencyCode, wallet.fiatCurrencyCode, denomination, denomName)
 }
 
+export const updateTxsFiat = (wallet: EdgeCurrencyWallet, currencyCode: string, txs: EdgeTransaction[]) => async (dispatch: Dispatch, getState: GetState) => {
+  const state = getState()
+  const exchangeDenom = getExchangeDenomination(state, wallet.currencyInfo.pluginId, currencyCode)
+  const { fiatCurrencyCode } = wallet
+
+  let promises: Array<Promise<void>> = []
+  for (const tx of txs) {
+    if (tx.metadata == null || tx.metadata.amountFiat === 0) {
```

Comment:
The EdgeMetadata type says amountFiat is optional so we should check for that case as well